packages/odie-client/src/components/send-message-input/index.tsx

```diff
@@ -20,6 +25,35 @@ export const OdieSendMessageButton = () => {
 	const isChatBusy = chat.status === 'loading' || chat.status === 'sending';
 	const [ isMessageSizeValid, setIsMessageSizeValid ] = useState( true );
 	const [ submitDisabled, setSubmitDisabled ] = useState( true );
+	const [ fileToUpload, setFileToUpload ] = useState< File | undefined >();
+	const onFilesDrop = ( files: File[] ) => {
+		const file = files?.[ 0 ];
+		if ( file && file.type.startsWith( 'image/' ) ) {
+			setFileToUpload( file );
+		}
+	};
+
+	const { zendeskClientId } = useSelect( ( select ) => {
+		const helpCenterSelect: HelpCenterSelect = select( HELP_CENTER_STORE );
+		return {
+			zendeskClientId: helpCenterSelect.getZendeskClientId(),
+		};
+	}, [] );
+	const onPaste = ( event: React.ClipboardEvent ) => {
+		const items = event.clipboardData.items;
+
+		for ( const item of items ) {
```

Comment:
We allow single files on drag and many on paste?

Reply:
No only one in both cases, I'll try to understand what we want.